docs/content/mongocxx-v3/installation/linux.md

```diff
@@ -111,7 +83,13 @@ cmake ..                                            \
     -DCMAKE_INSTALL_PREFIX=/usr/local
 ```
 
-### Step 5: Build and install the driver
+If CMake is unable to find an existing C driver installation, then it will
```

Comment:
Suggest removing this paragraph from each of the install instruction pages. I expect users following this tutorial will not need to install the C driver manually. I expect needing to install libmongoc separately is an advanced use case.